src/main/java/io/gingersnapproject/runtime/BackendListener.java

```diff
@@ -0,0 +1,46 @@
+package io.gingersnapproject.runtime;
+
+import java.net.URI;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import io.gingersnapproject.cdc.event.BackendFailedEvent;
+import io.gingersnapproject.cdc.event.BackendStartedEvent;
+import io.gingersnapproject.cdc.event.BackendStoppedEvent;
+
+import org.eclipse.microprofile.health.Liveness;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Liveness
```

Comment:
We need to add a `@Readiness` check which should return up/down once a connection has been established/broken between the db-syncer and both the Database and cache. I don't think the `BackendStartedEvent` and `BackendFailedEvent` is applicable to the Liveness check. If the Liveness probe fails, then k8s restarts the pod, but in this case restarting a pod won't help restore the backend or connector connection.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/